cmd/ddev/cmd/composer-create.go

```diff
@@ -27,25 +27,44 @@ var ComposerCreateCmd = &cobra.Command{
 web container. Projects will be installed to a temporary directory and moved to
 the Composer root directory after install.`,
 	Example: `ddev composer create drupal/recommended-project
-ddev composer create -y drupal/recommended-project
+ddev composer create drupal/recommended-project
 ddev composer create "typo3/cms-base-distribution:^10"
 ddev composer create drupal/recommended-project --no-install
 ddev composer create --repository=https://repo.magento.com/ magento/project-community-edition
 ddev composer create --prefer-dist --no-interaction --no-dev psr/log
-ddev composer create --preserve-flags --no-interaction psr/log
 `,
 	ValidArgsFunction: getComposerCompletionFunc(true),
 	Run: func(cmd *cobra.Command, _ []string) {
-		preserveFlags, _ := cmd.Flags().GetBool("preserve-flags")
-
 		// We only want to pass all flags and args to Composer
 		// cobra does not seem to allow direct access to everything predictably
-		osargs := []string{}
+		var osargs []string
 		if len(os.Args) > 3 {
 			osargs = os.Args[3:]
-			osargs = nodeps.RemoveItemFromSlice(osargs, "--yes")
-			osargs = nodeps.RemoveItemFromSlice(osargs, "-y")
-			osargs = nodeps.RemoveItemFromSlice(osargs, "--preserve-flags")
+			var expandedOsargs []string
+			for _, osarg := range osargs {
+				// If this is a combined short option like "-test", split it into "-t -e -s -t"
```

Comment:
Weird that we have to do this, when Cobra has these capabilities. I know there's *lots* of history in this code.

Reply:
I don't think we are using Cobra here, because `osargs = os.Args[3:]` at the top
In any case, there is always a way to improve it in the future.